plugin.json

```diff
@@ -21,12 +23,8 @@
 
     "dashboards" : [
         {
-            "name" : "Nginx Free",
+            "name" : "Nginx F/OSS",
             "layout" : "d-w=3&d-h=2&d-pad=5&d-bg=none&d-g-NGINX_ACTIVE_CONNECTIONS=0-0-1-1&d-g-NGINX_READING=2-1-1-1-t&d-g-NGINX_WRITING=2-1-1-1-b&d-g-NGINX_REQUESTS=1-0-1-1&d-g-NGINX_HANDLED=1-1-1-1-t&d-g-NGINX_NOT_HANDLED=1-1-1-1-b&d-g-NGINX_REQUESTS_PER_CONNECTION=2-0-1-1&d-g-NGINX_WAITING=0-1-1-1"
-        },
-        {
-            "name" : "Nginx Plus",
```

Comment:
we removed the nginx plus dashboard, but we are collecting the nginx plus metrics still, any reason to remove this?